db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -397,25 +397,25 @@ class NonBatchedOpsStressTest : public StressTest {
           iter->SeekToFirst();
           for (int i = 0; i < 5 && iter->Valid(); ++i, iter->Next()) {
           }
-        } else if (1 == rnd) {
+        } else if (1 == rnd && !FLAGS_auto_readahead_size) {
```

Comment:
Is there any way to ignore the auto_readahead_size option in FilePrefetchBuffer when Prev is called? I mean, its unlikely to be called in the Zippy case, but I don't know if its a good idea to leave that case unhandled.